packages/design-picker/package.json

```diff
@@ -42,13 +43,21 @@
 		"utility-types": "^3.10.0"
 	},
 	"devDependencies": {
+		"@automattic/calypso-build": "workspace:^",
 		"@automattic/calypso-typescript-config": "workspace:^",
 		"@testing-library/jest-dom": "^5.15.0",
 		"@testing-library/react": "^12.1.2",
+		"enzyme": "^3.11.0",
+		"jest": "^27.2.4",
 		"jest-canvas-mock": "^2.3.0",
+		"postcss": "^8.3.11",
 		"react": "^17.0.2",
 		"react-dom": "^17.0.2",
-		"typescript": "^4.4.4"
+		"react-native": "^0.66.2",
```

Comment:
Do you know why this was needed? I plan to remove it in https://github.com/Automattic/wp-calypso/pull/58228. I couldn't find any other use of `react-native` in the codebase -- maybe stale from when we still had it around?